blocks/header-nav-chain-block/themes/news.json

```diff
@@ -423,7 +423,8 @@
 				"components": {
 					"link": {
 						"width": "100%",
-						"padding": "var(--global-spacing-2) var(--global-spacing-5)",
+						"padding-top": "var(--global-spacing-2)",
+						"padding-right": "var(--global-spacing-5)",
```

Comment:
There should also be `"padding-bottom": "var(--global-spacing-2)",` and `"padding-left": "var(--global-spacing-5)",` here, since that's what the shorthand `padding` property was doing.